blockonomics.php

```diff
@@ -743,7 +758,15 @@ public function updateSettings()
 
     public function generateNewCallbackSecret()
     {
+        $this->setShopContextAll();
         $secret = md5(uniqid(rand(), true));
         Configuration::updateValue('BLOCKONOMICS_CALLBACK_SECRET', $secret);
     }
+
+    public function setShopContextAll()
```

Comment:
We could comment what this function does. Poor naming of Prestashops functions like `Shop::isFeatureActive` dont give enough context clue on what is happening.